core-root/navigation/Root.tsx

```diff
@@ -0,0 +1,50 @@
+import { BASE_HEADER_SCREEN_OPTIONS } from "@components/Navigation"
+import { withAlphaRegistration } from "@core-root/AlphaRegister"
+import { HomeView } from "@core-root/Home"
+import {
+  StaticParamList,
+  createStaticNavigation
+} from "@react-navigation/native"
+import { createNativeStackNavigator } from "@react-navigation/native-stack"
+import { EditEventNavigator } from "./EditEvent"
+import { eventDetailsScreens } from "./EventDetails"
+import { StyleSheet } from "react-native"
+
+const HomeScreen = withAlphaRegistration(() => (
+  <HomeView style={styles.screen} />
+))
+
+const Stack = createNativeStackNavigator({
+  screenOptions: BASE_HEADER_SCREEN_OPTIONS,
+  screens: {
+    home: {
+      options: { headerShown: false },
+      screen: HomeScreen
+    },
+    ...eventDetailsScreens()
+  },
+  groups: {
+    modals: {
+      screenOptions: { presentation: "modal", headerShown: false },
+      screens: { editEvent: EditEventNavigator }
+    }
+  }
+})
+
+const Navigation = createStaticNavigation(Stack)
+
+const linking = { prefixes: ["tifapp://"] }
+
+type RootStackParamList = StaticParamList<typeof Stack>
```

Comment:
createStaticNavigation? This is new to me, but when looking this up, apparently it's a different approach to navigation using a "static API". It also highlights deep linking; is that what we're using it for?

Reply:
It's just much more convenient than using the "dynamic" API, especially when it comes to the way that we share routes between navigators. We'll also be using it for deeplinking in the near future since it makes that much easier as well.